ethcore/Cargo.toml

```diff
@@ -110,3 +112,5 @@ test-heavy = []
 benches = []
 # Compile test helpers
 test-helpers = ["tempdir"]
+# Slow functionalities for evmbin or test only.
+evmbin-only = []
```

Comment:
Just a nitpick -- suggest to call it something more direct, or maybe just `to-pod-slow`, or `to-pod-full`.